### PR TITLE
Task link screens

### PR DESCRIPTION
**Pull request by alicemcneil**
Linked the screen together, created the Go Back button which take us back to the Splash screen, refactored Css and Html.

### CODE REVIEW

File: css/instructions.css [hunk 1]
```diff
@@ -1,17 +1,3 @@
-html {
```
Comment (snack-attack) [inline css/instructions.css:1]: delete this file if not using

File: game.html [hunk 2]
```diff
@@ -10,37 +10,70 @@
         <meta name="description" content="Mayden Academy JS Game">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/splash.css">
```
Comment (snack-attack) [inline game.html]: delete if not using

File: css/splash.css [hunk 3]
```diff
@@ -1,48 +1 @@
-.body {
```
Comment (snack-attack) [inline css/splash.css:1]: delete this file if not using

File: splash.html [hunk 4]
```diff
@@ -1,22 +1,126 @@
 <!DOCTYPE html>
```
Comment (snack-attack) [inline splash.html]: Rename this file to game.html if this is the only html file used in the game

File: splash.html [hunk 5]
```diff
@@ -1,22 +1,126 @@
...
         <div class="buttons">
             <h1 class="heading">whack-a-rat!</h1>
-            <button class="button">start</button>
-            <button class="button">instructions</button>
+            <button class="button start_button">start</button>
+            <button class="button instruction_button">instructions</button>
         </div>
+    </div>
+</div>
+
```
Comment (snack-attack) [inline splash.html]: unecessary whitespace in all these lines below

File: splash.html [hunk 6]
```diff
@@ -1,22 +1,126 @@
...
+            3. To let you get ready for hitting rats, the countdown timer will count 3...2...1...GO!<br>
+            4. Once a countdown is finished, rats will start randomly appear on their desks.<br>
+            5. Make rats disappear simply clicking on any part of them. Even whiskers are in the game >.<<br>
+            6. All rats you miss will stay for 1 second and then disappear themselves.<br>
+            7. Each rat you hit will bring you 1 point.<br>
+            8. You could see your score in a top right corner of a game screen. Don't look there though, concentrate on rats, because...<br>
+            9. Every 5 seconds the game will become more challenging and more rats will appear, try your best to whack them all, AND....<br>
+            10. Remember, you will only have 30 seconds to become a Whack-A-RAT world champion. Once 30 seconds are up, the game will stop and you will see your impressive final score and success rate.<br>
+            Thank you for reading.<br>
+            Now, let's whack some rats !</p>
```
Comment (snack-attack) [inline splash.html]: unnecessary white space after rats

File: splash.html [hunk 7]
```diff
@@ -1,22 +1,126 @@
...
+        <button class="back">Go Back!</button>
+    </div>
+
```
Comment (snack-attack) [inline splash.html]: delete extra whitespace

File: splash.html [hunk 8]
```diff
@@ -1,22 +1,126 @@
...
-    <title>whack-a-rat!</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Whack-a-Rat</title>
+    <meta name="description" content="Mayden Academy JS Game">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/game.css">
 </head>
-<body class="body">
```
Comment (snack-attack) [inline splash.html:11]: you need a body tag wrapped around all this code

File: splash.html [hunk 9]
```diff
@@ -1,22 +1,126 @@
...
+    </div>
+
+
+
+<script src="linked_pages.js"></script>
+
+
+</html>
 
-</body>
```
Comment (snack-attack) [inline splash.html:21]: close your body tag

File: splash.html [hunk 10]
```diff
@@ -1,22 +1,126 @@
...
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/game.css">
 </head>
-<body class="body">
+
+<!--SPLASH SCREEN-->
+
+<div class="splash">
     <div class="gamescreen">
         <img class="image" src="images/exploding.svg">
```
Comment (snack-attack) [inline splash.html]: there shouldn't be two images folder. Move the exploding img to the assets folder and delete the duplicated 'images' folder in the root dir

File: linked_pages.js [hunk 11]
```diff
@@ -0,0 +1,24 @@
+document.querySelector(".instruction_button").addEventListener('click', function () {
```
Comment (snack-attack) [inline linked_pages.js]: this query selector could be stored in a variable so you don't need to query twice

File: linked_pages.js [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+    document.querySelector('.instructions-container').classList.remove('no_show')
+})
+
+
+document.querySelector(".instruction_button").addEventListener('click', function () {
```
Comment (snack-attack) [inline linked_pages.js:4]: all these queries could be stored in variables as you're querying more than once

File: linked_pages.js [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+    document.querySelector('.splash').classList.add('no_show')
+})
+
```
Comment (snack-attack) [inline linked_pages.js:8]: unnecessary whitespace

File: linked_pages.js [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+document.querySelector(".start_button").addEventListener('click', function () {
+    document.querySelector('.splash').classList.add('no_show')
+})
+
+document.querySelector(".back").addEventListener('click', function () {
+    document.querySelector('.splash').classList.remove('no_show')
+    document.querySelector('.instructions-container').classList.add('no_show')
+
```
Comment (snack-attack) [inline linked_pages.js]: delete whitespace

File: splash.html (hunk 4 above)
Comment (alicemcneil) [reply splash.html]: now game.html

File: linked_pages.js (hunk 11 above)
Comment (alicemcneil) [reply linked_pages.js]: gotcha 👍

File: css/game.css [hunk 15]
```diff
@@ -51,3 +89,148 @@ header img {
...
+
+.table {
+    width: 83%;
+    position: relative;
+    z-index: 3;
+}
+
+.rat {
+    display: inline-block;
+    /*visibility: hidden;------->hides rat div */
```
Comment (snack-attack) [inline css/game.css]: delete commented code

File: css/game.css [hunk 16]
```diff
@@ -51,3 +89,148 @@ header img {
...
+    height: 800px;
+    padding: 20px;
+}
+
+.instructions-container h1 {
+    text-align: center;
+    font-family: 'Marker';
+}
+
+.instructions-container h1 {
```
Comment (snack-attack) [inline css/game.css]: this selector is a repeat of the one above - add this to the selector above

File: css/game.css [hunk 17]
```diff
@@ -22,23 +23,60 @@ html {
...
 
 body {
+    background: #b8c77c;
     font-family: 'Heebo';
-    font-size: 4rem;
+    font-size: 3rem;
 }
 
-.whiteboard {
+.gamescreen {
```
Comment (snack-attack) [inline css/game.css]: merge the selector here with .gamescreen on line 141
